skypy/halos/_colossus.py

```diff
@@ -203,3 +203,137 @@ def colossus_mf(redshift, model, mdef, m_min, m_max, sky_area, cosmology,
                                         cosmology, sigma8, ns, size, resolution)
 
     return z, m
+
+
+def concentration(mass, mdef, redshift, model, cosmology, sigma8, ns):
+    r'''Halo concentration calculator.
+
+    This function calculates halo concentration(s) using the model of c-M relation
+    available in colossus.
+
+    Parameters
+    ----------
+    mass : float or array_like
+        Spherical overdensity halo mass in units of solar mass/h, corresponding
```

Comment:
We had a debate about the use of `h` in units. We finally decided to work in units without `h`. 
Is this relevant for you here?